_devSpecs/Protein.html

```diff
@@ -60,20 +61,23 @@
   bsc_description: Disease associated to this protein, if any.
   marginality: Recommended
   cardinality: MANY
-  controlled_vocab: ""
+  controlled_vocab: Any suitable disease ontology
```

Comment:
Should we list some, e.g. Disease Ontology, OMIM, Orphanet, etc

Reply:
This is not a blocking issue for this PR. Now captured in issue https://github.com/BioSchemas/specifications/issues/355